apps/client/src/components/Layouts/BaseLayout.tsx

```diff
@@ -33,9 +32,9 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
             span={{
               small: 1,
               medium: 2,
-              big: checker ? 6 : 5,
-              large: checker ? 12 : 9,
-              xLarge: checker ? 12 : 9,
+              big: 5,
+              large: 9,
+              xLarge: 9,
```

Comment:
Oh wow, was the difference in columns actually not supposed to change?

Reply:
It didn't change